go/vt/vtgate/endtoend/row_count_test.go

```diff
@@ -24,13 +24,15 @@ import (
 	"github.com/stretchr/testify/require"
 
 	"vitess.io/vitess/go/mysql"
+	"vitess.io/vitess/go/test/endtoend/utils"
 )
 
 func TestRowCount(t *testing.T) {
 	ctx := context.Background()
 	conn, err := mysql.Connect(ctx, &vtParams)
 	require.NoError(t, err)
 	defer conn.Close()
+	utils.Exec(t, conn, "use ks")
```

Comment:
This use statement is now necessary because the number of keyspace is multiple. 3afc402